resources/assets/lib/follow-user-mapping-button.tsx

```diff
@@ -43,29 +53,40 @@ export default class FollowUserMappingButton extends React.Component<Props, Stat
   }
 
   render() {
-    if (currentUser.id == null || currentUser.id === this.props.userId) {
+    if (currentUser.id === this.props.userId && !this.props.alwaysVisible) {
       return null;
     }
 
-    const title = osu.trans(`follows.mapping.${this.state.follow ? 'to_0' : 'to_1'}`);
+    const currentUserProfile = currentUser.id === this.props.userId;
+    const notLoggedIn = currentUser.id == null;
+
+    let title = osu.trans(`follows.mapping.${this.state.follow ? 'to_0' : 'to_1'}`);
+    if (currentUserProfile || notLoggedIn) title = osu.trans(`follows.mapping.followers`);
```

Comment:
can be done at once with ternary operator `const title = someCheck ? osu.trans(...) : ....`